IntegrationTests/IntegrationTests.swift

```diff
@@ -42,10 +42,10 @@ class IntegrationTests: XCTestCase {
                 response = nil
             }
             XCTAssertNotNil(response)
-            XCTAssert(response?.enabled == true)
-            XCTAssertNotNil(response?.endpoints.ping)
-            XCTAssertNotNil(response?.endpoints.displayPermission)
-            XCTAssertNotNil(response?.endpoints.impression)
+            XCTAssert(response?.rolloutPercentage == 100)
```

Comment:
Can we assume that rollout will always be 100? Maybe just check if it's not nil or >0?

Reply:
Maybe or we can check only if `rolloutPercentage` is not nil?
Is it possible that `rolloutPercentage` be 0?

Reply:
Ah right is can be 0. I think checking for nil is enough